content/freeze.md

```diff
@@ -409,3 +412,5 @@ Example JavaScript (ECMAScript 6) code to check whether an account has Global Fr
 # See Also #
 
 [Gateway Bulletin GB-2014-02 New Feature: Balance Freeze](https://ripple.com/files/GB-2014-02.pdf)
+
+[//] # (Ugh, I hate that this PDF is all we have for "See Also." No other alternative that I can think of, though.)
```

Comment:
Sure, that's a good idea. My concern was more that we don't have a better place for the information in that PDF. But, like I said, I don't think there's really an alternative.